source/plugins/ovs-plugin.rst

```diff
@@ -364,6 +364,47 @@ To enable DPDK on VM deployments:
       dpdk-interface-model:
       <model type='virtio'/>
 
+Additional configurations on service offerings
```

Comment:
A title of something like:
Applying 'Additional Configurations' via Service Offerings
might be clearer, I misunderstood it the first time that I read it

Reply:
Done, thanks